tests/components/__snapshots__/autocomplete_dropdown.test.ts.snap

```diff
@@ -10,7 +10,18 @@ exports[`Functions autocomplete autocomplete simple snapshot with =S 1`] = `
     <div
       class="o-autocomplete-value text-truncate"
     >
-      SUM
+      <span
+        class="text-decoration-underline"
+        style="color: #9B359B;"
+      >
+        S
+      </span>
+      <span
+        style="color: undefined;"
```

Comment:
`"color: undefined;"` looks louche. `style` attribute should probably be omitted if there's no color